gatsby-node.js

```diff
@@ -5,42 +5,73 @@
  */
 const path = require('path');
 
-const ProgramPage = path.resolve('./src/components/masters/program/ProgramPage.jsx');
+const programPageType = 'pages.ProgramPage';
+const programListingTemplate = path.resolve('../src/components/masters/programs-list/ProgramListPage.jsx');
```

Comment:
nit: It might make sense to create a `templates` object:

```js
const templates = {
  programListPage: path.resolve('../src/components/masters/programs-list/ProgramListPage.jsx'),
  programPage: path.resolve('./src/components/masters/program/ProgramPage.jsx'),
};
```

And then:

```js
createPage({
  path: '/',
  component: templates.programListPage,
  context: { programs: allProgramsData },
});
```